submissions/tests/test_models.py

```diff
@@ -166,3 +170,52 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+
+class TestTeamSubmission(TestCase):
+    """
+    Test the TeamSubmission class
+    """
+    def test_create_team_submission(self):
+        user = TestTeamSubmission.create_user('user1')
+        ts = TeamSubmission.objects.create(
+            submitted_by=user,
+            team_id='t1',
+            course_id='c1',
+            item_id='i1',
+            attempt_number=0
+        )
+        # force evaluation of __str__ to ensure there are no issues with the class, since there
+        # isn't much specific to assert.
+        self.assertNotEqual(ts.__str__, None)
+
+    def test_create_duplicate_team_submission_not_allowed(self):
+        user = TestTeamSubmission.create_user('user1')
+        TeamSubmission.objects.create(
+            submitted_by=user,
+            team_id='t1',
+            course_id='c1',
+            item_id='i1',
+            attempt_number=0
+        )
+        with pytest.raises(DuplicateTeamSubmissionsError):
+            TeamSubmission.objects.create(
+                submitted_by=user,
+                team_id='t1',
+                course_id='c1',
+                item_id='i1',
+                attempt_number=1
+            )
+
+    @classmethod
```

Comment:
Nit: you can make this a `staticmethod` instead, since you're not using `cls`.  And then you'll be able to do `self.create_user()` in your test functions.